backend-lib/src/main/proto/proposal.proto

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2023 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+syntax = "proto3";
+package cash.z.wallet.sdk.ffi;
+
+// A data structure that describes the inputs to be consumed and outputs to
+// be produced in a proposed transaction.
+message Proposal {
```

Comment:
I'd prefer if [zcash/librustzcash#1060] could merge before we commit `proposal.proto` to this repository, since there are substantial changes there.

Reply:
As the PR in question has now been merged, I'll rebase this PR to include it.